src/layup/convert.py

```diff
@@ -0,0 +1,92 @@
+import os
+import math
+from pathlib import Path
+from typing import Literal
+
+from concurrent.futures import ProcessPoolExecutor
+
+from layup.file_io.CSVReader import CSVDataReader
+from layup.file_io.HDF5Reader import HDF5DataReader
+from layup.file_io.file_output import write_csv, write_hdf5
+
+
+def process_chunk(data):
+    pass
+
+
+def convert(
```

Comment:
so I think we should this run_convert or something like that. I expect that orbit fitting will need converting to the ecliptic plane reference plane and also allowing convert to be the name of the function that does the orbit converting might be cleaner for those that need to use that if they have mixed types and directly using th python API

Reply:
Would it make sense to call this `convert_orbit_format` or something similar? If `run_convert` makes sense, we can use that, but the name didn't seem immediately intuitive about how it would distinguish between the different converting cases.

Reply:
I believe that this should be resolved as of the last commit from @wilsonbb